cognite/transformations_cli/commands/deploy/transformations_api.py

```diff
@@ -99,19 +99,23 @@ def to_destination(destination: DestinationConfigType) -> TransformationDestinat
         return DataModelInstances(
             destination.model_external_id, destination.space_external_id, destination.instance_space_external_id
         )
-    elif isinstance(destination, InstanceNodesDestinationConfig):
-        view = None
-        if destination.view:
-            view = ViewInfo(destination.view.space, destination.view.external_id, destination.view.version)
-        return InstanceNodes(view, destination.instance_space)
-    elif isinstance(destination, InstanceEdgesDestinationConfig):
-        view = None
-        if destination.view:
-            view = ViewInfo(destination.view.space, destination.view.external_id, destination.view.version)
-        edge_type = None
-        if destination.edge_type:
-            edge_type = EdgeType(destination.edge_type.space, destination.edge_type.external_id)
-        return InstanceEdges(view, destination.instance_space, edge_type)
+    elif isinstance(destination, InstanceNodesDestinationConfig) or isinstance(
+        destination, InstanceEdgesDestinationConfig
+    ):
+        if destination.type == InstanceNodesDestinationConfig.type:
+            view = None
+            if destination.view:
+                view = ViewInfo(destination.view.space, destination.view.external_id, destination.view.version)
+            return InstanceNodes(view, destination.instance_space)
+        elif destination.type == InstanceEdgesDestinationConfig.type:
+            view = None
+            if destination.view:
+                view = ViewInfo(destination.view.space, destination.view.external_id, destination.view.version)
+            edge_type = None
+            # This is a hack, we should have better fix since destination is still InstanceNodesDestinationConfig here...
+            if isinstance(destination, InstanceEdgesDestinationConfig) and destination.edge_type:
```

Comment:
does the type checker complain if you remove the `isinstance` condition?

Reply:
Don't see anything so far, at least the CI doesn't complain 😂 
How can we run the type checker?